src/shared/nt-core/withLoader.styl

```diff
@@ -0,0 +1,38 @@
+@require('../nt-design/colorsets')
+
+l-size = 5em
+l-border = 5px
+l-spin-time = 1000ms
```

Comment:
Fixed

P.S. what's about variables in "colorsets.styl" file? should I rename all of them?